topics/introduction/tutorials/python-plotting/tutorial.md

```diff
@@ -0,0 +1,281 @@
+
+    ```python
+    exercise_data = data.groupby(['Chromosome', 'Strand']).agg(mean_pvalue = ('P-value', 'mean')).reset_index()
+    
+    colors = {'+':'red', '-':'blue'}
+    plt.scatter(x = exercise_data['Chromosome'], y = exercise_data['mean_pvalue'], c = exercise_data['Strand'].map(colors))
+    plt.ylabel('Average P-value')
+    plt.xlabel('Chromosome')
+    ```
+
+![Figure15](../../images/python-plotting/Figure15_Exercise_plot.png)
+
+## Making your plots accessible
+Whenever you are generating plots to go into a paper or a presentation, there are a few things you can do to make sure that everyone can understand your plots.
+
+Always make sure your text is large enough to read. Use the `fontsize` parameter in `xlabel`, `ylabel`, `title`, and `legend`, and `tick_params` with `labelsize` to increase the text size of the numbers on your axes.
+Similarly, you should make your graph elements easy to see. Use `s` to increase the size of your scatterplot markers and `linewidth` to increase the sizes of your plot lines.
+Using `color` (and nothing else) to distinguish between different plot elements will make your plots unreadable to anyone who is colorblind, or who happens to have a black-and-white office printer. For lines, the `linestyle` parameter lets you use different types of lines. For scatterplots, `marker` lets you change the shape of your points.
+
+
+
+# Conclusion
+
+This tutorial aims to serve as an introduction to plotting using the Python programming language.
+
+## Key points
+
+* Python has many libraries offering a variety of capabilities, which makes it popular for beginners, as well as, more experienced users
+
+* You can use scientific libraries like Matplotlib to perform exploratory data analysis.
+
+
+
+# References
+
+1. Wendi Bacon, Mehmet Tekman, 2021, Trajectory Analysis using Python (Jupyter Notebook) in Galaxy (Galaxy Training Materials). https://training.galaxyproject.org/training-material/topics/transcriptomics/tutorials/scrna-JUPYTER-trajectories/tutorial.html Online; accessed Wed Sep 01 2021
+2. Batut et al., 2018, Community-Driven Data Analysis Training for Biology Cell Systems [10.1016/j.cels.2018.05.012
+3.  Azalee Bostroem, Trevor Bekolay, and Valentina Staneva (eds): "Software Carpentry: Programming with Python."  Version 2016.06, June
+2016, https://github.com/swcarpentry/python-novice-inflammation, 10.5281/zenodo.57492.
+4. Allen Lee, Nathan Moore, Sourav Singh, Olav Vahtras (eds). Software Carpentry: Plotting and Programming in Python. http://github.com/swcarpentry/python-novice-plotting, 2018.
+5. Bérénice Batut, Mallory Freeberg, Mo Heydarian, Anika Erxleben, Pavankumar Videm, Clemens Blank, Maria Doyle, Nicola Soranzo, Peter van Heusden, 2021 Reference-based RNA-Seq data analysis (Galaxy Training Materials). https://training.galaxyproject.org/training-material/topics/transcriptomics/tutorials/ref-based/tutorial.html Online; accessed Wed Sep 01 2021
+
```

Comment:
```suggestion
```

Could you move these into the appropriate `tutorial.bib` file and then include `{% cite ... %}` blocks where relevant? https://doi2bib.org/ is a great site to help make that file.